### PR TITLE
Update French Translations

### PR DESCRIPTION
**Pull request by Elgin675**
**When merged this pull request will:**
- Title

### CODE REVIEW

File: addons/medical/stringtable.xml [hunk 1]
```diff
@@ -46,6 +46,7 @@
         <Key ID="STR_ACE_Medical_spontaneousWakeUpEpinephrineBoost_DisplayName">
             <English>Epinephrine Wake Up Chance Boost</English>
             <French>L'épinéphrine augmente les chances de réveil</French>
+            <Polish>Zwiększenie szansy na wybudzenie po podaniu epinefryny</Polish>
```
Comment (dedmen) [inline addons/medical/stringtable.xml]: > Translations: French

> <Polish>

🤔

Comment (Elgin675) [reply addons/medical/stringtable.xml]: @dedmen Apparently, it happened when I was merged master :
https://github.com/acemod/ACE3/blob/master/addons/medical/stringtable.xml#L47L48

What am I supposed to do now ?

Comment (PabstMirror) [reply addons/medical/stringtable.xml]: I don't think it's a problem, doesn't show on PR diff

Comment (PabstMirror) [reply addons/medical/stringtable.xml]: `Zwiększenie szansy na wybudzenie po podaniu epinefryny` is same as current master current

**Pull request merged**